<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsWindow</class>
 <widget class="QWidget" name="NewsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>72</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="url1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>URL 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="url2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>641</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>URL 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="url5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>URL 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="url3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>URL 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="url4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>641</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>URL 4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="category">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter search command:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
